build.gradle

```diff
@@ -90,14 +90,18 @@ subprojects {
 
   dependencies {
     compile 'org.slf4j:slf4j-api'
-    compile 'com.google.guava:guava'
+    compile('com.google.guava:guava') {
+      // may be LGPL - use ALv2 findbugs-annotations instead
+      exclude group: 'com.google.code.findbugs'
+    }
+    compile 'com.github.stephenc.findbugs:findbugs-annotations:1.3.9-1'
```

Comment:
This was needed because the version from Guava may or may not be LGPL.

See:
* https://github.com/stephenc/findbugs-annotations
* https://github.com/findbugsproject/findbugs/issues/128